.ci/test-r-package-windows.ps1

```diff
@@ -176,16 +176,7 @@ Remove-Item "$env:RTOOLS_MINGW_BIN/cmake.exe" -Force -ErrorAction Ignore
 Write-Output "Done installing CMake"
 
 Write-Output "Installing dependencies"
-$packages = -join @(
-    "c('data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'processx', 'R6', 'RhpcBLASctl', 'testthat'), ",
-    "dependencies = c('Imports', 'Depends', 'LinkingTo')"
-)
-$params = -join @(
-    "options(install.packages.check.source = 'no'); ",
-    "install.packages($packages, repos = '$env:CRAN_MIRROR', type = 'binary', ",
-    "lib = '$env:R_LIB_PATH', Ncpus = parallel::detectCores())"
-)
-Invoke-R-Code-Redirect-Stderr $params ; Assert-Output $?
+Rscript.exe --vanilla ".ci/install-r-deps.R" --build --test ; Assert-Output $?
```

Comment:
Seem that `processx` was lost.

https://github.com/microsoft/LightGBM/blob/c9de57b09cf9293213fa41b423e2409ebd9ae5c1/R-package/inst/make-r-def.R#L40-L42

Reply:
Fixed in https://github.com/microsoft/LightGBM/pull/6808/commits/dae117f3d94d6c49873e6759b4da1b0a8bd7ea0e.

```pwsh
Rscript.exe --vanilla ".ci/install-r-deps.R" --build --include=processx --test ; Assert-Output $?
```